cmd/createcluster.go

```diff
@@ -84,8 +86,8 @@ func newCreateClusterCmd(runFunc func(io.Writer, clusterConfig) error) *cobra.Co
 func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.StringVar(&config.Name, "name", "", "Optional cosmetic cluster name")
 	flags.StringVar(&config.ClusterDir, "cluster-dir", ".charon/cluster", "The target folder to create the cluster in.")
-	flags.IntVarP(&config.NumNodes, "nodes", "n", 4, "The number of charon nodes in the cluster.")
-	flags.IntVarP(&config.Threshold, "threshold", "t", 3, "The threshold required for signature reconstruction. Minimum is n-(ceil(n/3)-1).")
+	flags.IntVarP(&config.NumNodes, "nodes", "n", defaultNumNodes, "The number of charon nodes in the cluster.")
```

Comment:
update flag comments:
`nodes: The number of charon nodes in the cluster. Minimum is 4.`
`threshold. The threshold required for signature reconstruction. Strongly suggest leaving empty to defaults to ceil(n*2/3).`